File: LeetCode/Python/Easy/2540-Minimum-Common-Value/solution.py
class Solution:
    def getCommon(self, nums1: List[int], nums2: List[int]) -> int:
        result=-1

        n=len(nums1)

        m=len(nums2)

        i=0

        j=0

        while i < n and j < m:
            if nums1[i] < nums2[j]:
                i+=1
            elif nums1[i] > nums2[j]:
                j+=1
            else:
                result=nums1[i]

                break

        return result

File: LeetCode/Rust/Easy/2540-Minimum-Common-Value/solution.rs
impl Solution
{
    pub fn get_common(nums1: Vec<i32>, nums2: Vec<i32>) -> i32
    {
        let mut result: i32=-1;

        let n: usize=nums1.len();

        let m: usize=nums2.len();

        let mut i: usize=0;

        let mut j: usize=0;

        while i < n && j < m
        {
            if nums1[i] < nums2[j]
            {
                i+=1;
            }
            else if nums1[i] > nums2[j]
            {
                j+=1;
            }
            else
            {
                result=nums1[i];

                break;
            }
        }

        result
    }
}